### PR TITLE
DSDEGP-1710: get automated tests working

### PR DESCRIPTION
**Pull request by dags**
### Description
Updated test to avoid failures while improving Travis build.
----

### Checklist
Dsdegp 1710

### CODE REVIEW

File: src/app/app.component.spec.ts [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { SharedModule } from '@app/shared';
+import { CoreModule } from '@app/core';
+import { FileManagerModule } from '@app/file-manager';
+import { SettingsModule } from './settings';
+import { NgxElectronModule } from 'ngx-electron';
+import { RouterTestingModule } from '@angular/router/testing';
+
 describe('AppComponent', () => {
   beforeEach(async(() => {
```
Comment (pshapiro4broad) [inline src/app/app.component.spec.ts]: Is `async()` needed here, and below in the `it()` clauses? Other examples I've seen don't use this construct.

Comment (dgil-olx) [reply src/app/app.component.spec.ts]: Spec files are created by default by Angular CLI tool so I think follows Angular best practices. I need to do some research before being able to confirm it is a bad practice. Could we put this on testing technical debt ?

Comment (pshapiro4broad) [reply src/app/app.component.spec.ts]: Yes that would be OK

Comment (dags) [reply src/app/app.component.spec.ts]: Spec files are created by default by Angular CLI tool so I think follows Angular best practices. I need to do some research before being able to confirm it is a bad practice. Could we put this on testing technical debt ?

**Pull request merged**